### PR TITLE
GH-41 Add command cooldowns

### PR DESCRIPTION
**Pull request by Rollczi**
Releted: #41

## Summary by CodeRabbit

- **New Features**
	- Introduced command cooldown functionality to enhance user experience and command management.
	- Added configuration options for command cooldowns via a new commands.yml file.

- **Improvements**
	- Enhanced user feedback through formatted messages related to command cooldowns.

- **Bug Fixes**
	- Resolved issues with command execution timing, ensuring smoother interactions.

### CODE REVIEW

File: eternaleconomy-core/src/main/java/com/eternalcode/economy/config/implementation/CommandsConfig.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import eu.okaeri.configs.OkaeriConfig;
+import eu.okaeri.configs.annotation.Comment;
+
+import java.util.Map;
+
+public class CommandsConfig extends OkaeriConfig {
+
+    @Comment("Cooldowns for commands")
+    public Map<String, CommandCooldownConfig> cooldowns = Map.of(
+        "pay", new CommandCooldownConfig()
```
Comment (CitralFlo) [inline eternaleconomy-core/src/main/java/com/eternalcode/economy/config/implementation/CommandsConfig.java:13]: I like possible expansions to that 👍

File: eternaleconomy-core/src/main/java/com/eternalcode/economy/command/cooldown/CommandCooldownConfig.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import com.eternalcode.multification.notice.Notice;
+import eu.okaeri.configs.OkaeriConfig;
+
+import java.time.Duration;
+
+public class CommandCooldownConfig extends OkaeriConfig {
+    public Duration duration = Duration.ofSeconds(5);
+    public String bypassPermission = "eternaleconomy.player.pay.bypass";
+    public Notice message = Notice.builder()
+        .chat("<b><gradient:#00FFA2:#34AE00>ECONOMY</gradient></b> <dark_gray>➤</dark_gray> "
```
Comment (CitralFlo) [inline eternaleconomy-core/src/main/java/com/eternalcode/economy/command/cooldown/CommandCooldownConfig.java:12]: maybe make a template for that to use in other notifications?? like static preset

**Review by CitralFlo — APPROVED** [review]

File: eternaleconomy-core/src/main/java/com/eternalcode/economy/command/cooldown/CommandCooldownConfig.java (hunk 2 above)
Comment (Rollczi) [reply eternaleconomy-core/src/main/java/com/eternalcode/economy/command/cooldown/CommandCooldownConfig.java:15]: unrelated. #44

**Comment by vLuckyyy** [comment]
@coderabbitai full review

**Review by igoyek — APPROVED** [review]

**Pull request merged**